analyzer/pubclient/pubclient.go

```diff
@@ -0,0 +1,92 @@
+package pubclient
```

Comment:
I'd include a package comment that mentions this is to be used for connecting to untrusted URLs

Reply:
ok

Reply:
added